demos/blockfactory/app_controller.js

```diff
@@ -5,15 +5,14 @@
  */
 
 /**
- * @fileoverview The AppController Class brings together the Block
- * Factory, Block Library, and Block Exporter functionality into a single web
- * app.
- *
+ * Controller for the Blockly Factory
+ * @constructor
  */
 
 /**
- * Controller for the Blockly Factory
- * @constructor
+ * @fileoverview The AppController Class brings together the Block
+ * Factory, Block Library, and Block Exporter functionality into a single web
+ * app.
```

Comment:
This appears to be overzealous application of a script.  We should revert.

Reply:
(Just this one file, not the whole PR!)

Reply:
Oh, that's weird.  I thought this swap was in the other direction.